### PR TITLE
PFM-64 Add basic edit profile page

### PR DESCRIPTION
**Pull request by SamDudley**

### CODE REVIEW

File: peoplefinder/models.py [hunk 1]
```diff
@@ -2,13 +2,47 @@
 from django.urls import reverse
 
 
+class Country(models.Model):
+    DEFAULT_CODE = "GB"
+
+    code = models.CharField(max_length=2, primary_key=True)
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
```
Comment (SamDudley) [inline peoplefinder/models.py:11]: types

File: peoplefinder/models.py [hunk 2]
```diff
@@ -2,13 +2,47 @@
...
+        return self.name
+
+
+class Workday(models.Model):
+    code = models.CharField(max_length=3, primary_key=True)
+    name = models.CharField(max_length=9)
+
+    def __str__(self):
```
Comment (SamDudley) [inline peoplefinder/models.py:19]: types

File: peoplefinder/templates/peoplefinder/components/form.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{% for field in form %}
```
Comment (SamDudley) [inline peoplefinder/templates/peoplefinder/components/form.html:1]: delete file as it's not being used

File: peoplefinder/migrations/0003_person_pronouns_squashed_0008_auto_20210302_0840.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
+# Generated by Django 3.1.6 on 2021-03-02 08:53
```
Comment (SamDudley) [inline peoplefinder/migrations/0003_person_pronouns_squashed_0008_auto_20210302_0840.py:1]: squash migrations sensibly

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**